gluten-substrait/src/main/scala/org/apache/gluten/expression/ExpressionConverter.scala

```diff
@@ -164,6 +164,32 @@ object ExpressionConverter extends SQLConfHelper with Logging {
                 child.map(replaceWithExpressionTransformer0(_, attributeSeq, expressionsMap)),
                 i)
           }
+        } else if (objectName.endsWith("CharVarcharCodegenUtils")) {
```

Comment:
Please extract the logic for case i: StaticInvoke => to a new function, here will be much code